feg/protos/s6a_proxy.proto

```diff
@@ -217,9 +217,9 @@ message UpdateLocationAnswer {
 
     message AggregatedMaximumBitrate {
         // Maximum uplink bitrate
-        uint32 max_bandwidth_ul = 1;
+        uint64 max_bandwidth_ul = 1;
```

Comment:
we also have the same message definition on `apn.proto` file, which are used on sessiond and subscriberdb message requests. we should update those as well or look into consolidating both into one, @uri200 thoughts?

Reply:
I agree. That should be changed too. 
It would be good to modify a lte integ test to use those values and see if something breaks.